src/testdrive/src/format/avro.rs

```diff
@@ -220,10 +217,471 @@ pub fn from_json(json: &JsonValue, schema: SchemaNode) -> Result<Value, String>
+        (JsonValue::Object(_), SchemaPiece::Long) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Float) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Double) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Date) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::TimestampMilli) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::TimestampMicro) => todo!(),
+        (
+            JsonValue::Object(_),
+            SchemaPiece::Decimal {
+                precision,
+                scale,
+                fixed_size,
+            },
+        ) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Bytes) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::String) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Json) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Uuid) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Array(_)) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Union(_)) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveIntTsMilli) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveIntTsMicro) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveDateTimestamp) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveIntLong) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveIntFloat) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveIntDouble) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveLongFloat) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveLongDouble) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveFloatDouble) => todo!(),
+        (
+            JsonValue::Object(_),
+            SchemaPiece::ResolveConcreteUnion {
+                index,
+                inner,
+                n_reader_variants,
+                reader_null_variant,
+            },
+        ) => todo!(),
+        (
+            JsonValue::Object(_),
+            SchemaPiece::ResolveUnionUnion {
+                permutation,
+                n_reader_variants,
+                reader_null_variant,
+            },
+        ) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::ResolveUnionConcrete { index, inner }) => todo!(),
+        (
+            JsonValue::Object(_),
+            SchemaPiece::Enum {
+                doc,
+                symbols,
+                default_idx,
+            },
+        ) => todo!(),
+        (JsonValue::Object(_), SchemaPiece::Fixed { size }) => todo!(),
+        (
+            JsonValue::Object(_),
+            SchemaPiece::ResolveRecord {
+                defaults,
+                fields,
+                n_reader_fields,
+            },
+        ) => todo!(),
+        (
+            JsonValue::Object(_),
+            SchemaPiece::ResolveEnum {
+                doc,
+                symbols,
+                default,
+            },
+        ) => todo!(),
```

Comment:
What happened here? 🤔

Reply:
whaaaaaat. I imagine this was some rust-analyzer magic gone wild, I'll undo.